Makefile.toml

```diff
@@ -0,0 +1,5 @@
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
+
+[tasks.wasm-build]
+# see wasm/Makefile.toml for task definition
```

Comment:
Can you remove this file and instead use a deno.json with tasks? We'd rather not have to add `cargo install cargo-make` to the setup instructions.

Reply:
I tried `wasmbuild` but it only generates a single JS file that gets instantiated which felt like the old way to generate things given https://github.com/rustwasm/wasm-bindgen/issues/4287

I'm not sure if the plan is to update `wasmbuild` in some way (if yes, we should create an issue in `wasmbuild` so we can track it) or if the plan is to keep things as-is